cocotb/share/makefiles/simulators/Makefile.activehdl

```diff
@@ -59,14 +59,14 @@ endif
 	@echo "endsim" >> $@
 
 $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/runsim.do $(CUSTOM_COMPILE_DEPS) $(CUSTOM_SIM_DEPS)
-	-@rm -f $(COCOTB_RESULTS_FILE)
+	$(RM) $(COCOTB_RESULTS_FILE)
```

Comment:
-f is implied, I assume?

Reply:
Yes, `$(RM)` defaults to `rm -f` (see commit message).